automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UISpec.scala

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsde.firecloud.fixture
+
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.{Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.fixture._
+import org.broadinstitute.dsde.workbench.service.test.{RandomUtil, WebBrowserSpec}
+import org.scalatest.{BeforeAndAfterAll, Matchers, Outcome, fixture}
+
+import scala.util.Try
+
+
+abstract class UISpec extends fixture.FreeSpec with BeforeAndAfterAll with BillingFixtures with Matchers
```

Comment:
Nit: I think `UISpec` is too general of a name for the very specific thing this class is doing. I'm having trouble coming up with a good alternative, though... maybe `BillingFixtureSpec`?